test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/TypeEnumeratorTests.cs

```diff
@@ -177,84 +165,81 @@ public void GetTestsShouldNotReturnBaseTestMethodsFromAnotherAssemblyByConfigura
 
         var tests = typeEnumerator.Enumerate(out _warnings);
 
-        Assert.IsNotNull(tests);
-        Assert.IsTrue(
+        Verify(tests is not null);
+        Verify(
             tests.All(t => t.TestMethod.Name != "BaseTestMethod"),
             "DummyDerivedFromRemoteTestClass inherits DummyRemoteBaseTestClass from different assembly. BestTestMethod from DummyRemoteBaseTestClass should not be discovered when RunSettings MSTestV2 specifies EnableBaseClassTestMethodsFromOtherAssemblies = false.");
```

Comment:
Please move this arg to be a comment before the `Verify`. It's currently going in the `expression` param which we don't want to be filled by "user".